app/models/auction.rb

```diff
@@ -1,11 +1,29 @@
 class Auction < ActiveRecord::Base
   has_many :bids
   has_many :bidders, through: :bids
-  enum result: {not_applicable: 0, accepted: 1, rejected: 2}
-  enum type: {single_bid: 0, multi_bid: 1}
-  enum awardee_paid_status: {not_paid: 0, paid: 1}
-  enum published: {unpublished: 0, published: 1}
+  enum result: { not_applicable: 0, accepted: 1, rejected: 2 }
```

Comment:
I like them as enums (get free methods like `not_applicable?`) -- what do you see as the benefit of strings?